addons/web/static/src/scss/bootstrap_overridden.scss

```diff
@@ -98,7 +99,7 @@ $grid-gutter-width: $o-horizontal-padding * 2 !default;
 //
 // Define common padding and border radius sizes and more.
 
-$border-color: $o-border-color!default;
+$border-color: $o-border-color !default;
```

Comment:
Unrelated to this PR.
The change itself is good, but it's something we usually do during refactoring or code optimization tasks.